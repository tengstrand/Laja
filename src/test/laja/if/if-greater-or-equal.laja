#write "target/test-output/if-greater-or-equal.out"
  #set (x = 1)
  #if (x >= 1)
x is greater or equal to 1
  #end
  #if (x >= 2)
x is greater or equal to 2
  #else
x is not greater or equal to 2
  #end
#end