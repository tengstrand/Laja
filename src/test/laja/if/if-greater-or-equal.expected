x is greater or equal to 1
x is not greater or equal to 2
